python/pyspark/sql/utils.py

```diff
@@ -234,3 +236,19 @@ def wrapped(*args: Any, **kwargs: Any) -> Any:
         return f(*args, **kwargs)
 
     return cast(FuncT, wrapped)
+
+
+def pyspark_column_op(func_name: str) -> Callable[..., "SeriesOrIndex"]:
+    """
+    Wrapper function for column_op to get proper Column class.
+    """
+    from pyspark.pandas.base import column_op
+    from pyspark.sql.column import Column as PySparkColumn
+
+    if is_remote():
+        from pyspark.sql.connect.column import Column as ConnectColumn
+
+        Column = ConnectColumn
+    else:
+        Column = PySparkColumn  # type: ignore[assignment]
+    return column_op(getattr(Column, func_name))
```

Comment:
> Can we get a one utility to get the column class at least? e.g., utils.pyspark_column_op("__lt__")
@HyukjinKwon just added util function here. If it looks good, will apply this change to other PRs as well.